CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ project(ratslam_ros)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake)
 
+# Enable Cxx11 support
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
Agree! I wanted to replace raw pointers by smart ones in near future. Will take your snippet! Thanks!